threestudio/systems/zero123.py

```diff
@@ -67,20 +68,21 @@ def training_substep(self, batch, batch_idx, guidance: str):
             # claforte: surely there's a cleaner way to get batch size
             bs = batch["rays_o"].shape[0]
 
-            bg_color = torch.rand(bs, 3).to(self.device)  # claforte: use dtype
+            bg_color = torch.ones(bs, 3).to(self.device)
+            # bg_color = torch.rand(bs, 3).to(self.device)  # claforte: use dtype
 
-            # Override 50% of the bgcolors with white.
-            # This results in better predictions with Zero123,
-            # since it was trained with a constant white background.
-            white = torch.ones(bs, 3).to(self.device)
+            # # Override 50% of the bgcolors with white.
+            # # This results in better predictions with Zero123,
+            # # since it was trained with a constant white background.
+            # white = torch.ones(bs, 3).to(self.device)
```

Comment:
You disabled the "replace bg color by white 50% of the time"... but are you sure that this improves convergence?

Reply:
In our case, it doesn't affect the background at all because our bg is always white.
I can re-enable it if you think that's better?